.github/workflows/ci.yml

```
@@ -165,32 +165,41 @@ jobs:
           echo "🔢 Setting version based on deployment environment..."
           set -Eeuo pipefail
           
+          # Function to safely escape shell input
+          escape_shell_input() {
+            local input="$1"
+            # Remove potentially dangerous characters and limit length
+            echo "$input" | sed 's/[;&|`$(){}]/\\&/g' | cut -c1-200
+          }
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)